INSTALL.org

```diff
@@ -291,16 +291,12 @@ your R session:
 
 #+begin_src R :results output :session :exports both
 library(arrow)
-arrow_available()
```

Comment:
Unrelated, but this isn't needed anymore: https://github.com/apache/arrow/blob/main/r/R/arrow-info.R#L78-L81